fellow-submissions/laisha-calalpa/style.css

```diff
@@ -0,0 +1,65 @@
+html{
+    box-sizing: border-box;
+}
+*{
+    box-sizing:inherit;
+}
+
+body{
+    font-family: sans-serif;
+    display: flex;
```

Comment:
This is interesting. Why did you decide to flex the entire body? I have a feeling that this may have been unnecessary.